app/views/pages/home.html.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Lavamae</title>
+</head>
+<body>
+Home page
+</body>
+</html>
```

Comment:
you shouldn't need the entire html structure since it will be imported into `views/layouts/application.html.erb`